projects/storefrontapp-e2e-cypress/cypress/sample-data/powertools-checkout-flow.ts

```diff
@@ -0,0 +1,7 @@
+import { SampleUser, user } from './checkout-flow';
+
+export const variantUser: SampleUser = {
+  ...user,
+  password: '12341234',
+  email: 'marie.dubois@rustic-hw.com',
```

Comment:
B2B users tend to lock-up because of too many failed login attempts. 
Test case may fail randomly because of it. 
I strongly encourage the team to generate b2b users for e2e tests.